src/pages/index.js

```diff
@@ -322,3 +328,15 @@ deleteForm.addEventListener("submit", handleDeleteSubmit);
 editFormElement.addEventListener("submit", handleEditFormSubmit);
 cardForm.addEventListener("submit", handleAddCardSubmit);
 avatarForm.addEventListener("submit", handleAvatarSubmit);
+document.addEventListener("keydown", (evt) => {
+  const openModalElement = document.querySelector(".modal_opened");
+  if (openModalElement && evt.key === "Escape") {
+    closeModal(openModalElement);
+  }
+});
+
+document.addEventListener("click", (evt) => {
+  if (evt.target === document.querySelector(".modal_opened")) {
+    closeModal(document.querySelector(".modal_opened"));
```

Comment:
* And the `overlay` handler was correct in your project 6. Please, have a look at your code there https://snipboard.io/dJAVWa.jpg